draft-ietf-quic-transport.md

```diff
@@ -2714,13 +2714,19 @@ but not the UDP or IP header.
 
 The initial client packet SHOULD be padded to exactly 1200 octets unless the
 client has a reasonable assurance that the PMTU is larger.  Sending a packet of
-this size ensures that the network path supports an MTU of this size and helps
-reduce the amplitude of amplification attacks caused by server responses toward
-an unverified client address.
+this size ensures that the network path from the client to the server supports
+an MTU of this size and helps reduce the amplitude of amplification attacks
+caused by server responses toward an unverified client address.
 
 Servers MUST ignore an initial plaintext packet from a client if its total size
 is less than 1200 octets.
 
+Similarly, servers MUST ensure that the first handshake packet they send to
```

Comment:
Use capital H here and maybe be more explicit.  A Version Negotiation or Retry packet shouldn't be larger, and in fact should be kept small.  (We can't pad Version Negotiation anyway, except with greasing values, which isn't much use.)